miso-web/src/main/resources/sql/db-config.xml

```diff
@@ -190,6 +190,7 @@
     <property name="sampleQcDAO" ref="sqlSampleQCDAO"/>
     <property name="securityProfileDAO" ref="sqlSecurityProfileDAO"/>
     <property name="jdbcTemplate" ref="interfaceTemplate"/>
+    <property name="autoGenerateIdentificationBarcodes"><value>${miso.autoGenerateIdentificationBarcodes}</value></property>

```

Comment:
In the db4o config it looks like the db4oThingDAO beans are all commented out, so updating it there wouldn't do much.